src/plugins/controls/public/controls/data_controls/range_slider/components/range_slider_control.tsx

```diff
@@ -212,13 +212,11 @@ export const RangeSliderControl: FC<Props> = ({
                 content={RangeSliderStrings.control.getInvalidSelectionWarningLabel()}
                 delay="long"
               >
-                <EuiToken
+                <EuiIcon
                   tabIndex={0}
-                  iconType="alert"
+                  type="warningFilled"
                   size="s"
-                  color="euiColorVis5"
-                  shape="square"
-                  fill="dark"
+                  color="warning"
```

Comment:
Why the switch from `Token` to `Icon`?

Reply:
We did the token so that it matches the tokens used on the options list control (for both the warning **and** the selections count badge). These were designs previously approved, so I don't think it's right to make the change in this PR :) We should keep design changes minimal and only change colours IMO